includes/class-wc-calypso-bridge-payments.php

```diff
@@ -105,21 +109,23 @@ public function register_payments_welcome_page() {
 		// nav is enabled. The new nav disabled everything, except the 'WooCommerce' menu.
 		// We need to register this menu via add_menu_page so that it doesn't become a child of
 		// WooCommerce menu.
-		add_menu_page(
-			__( 'Payments', 'wc-calypso-bridge' ),
-			__( 'Payments', 'wc-calypso-bridge' ),
-			'view_woocommerce_reports',
-			'admin.php?page=wc-admin&path=/payments-welcome',
-			null,
-			null,
-			'55.7' // After WooCommerce & Product menu items.
-		);
+		if ( 'yes' === get_option('woocommerce_navigation_enabled', 'no') ) {
```

Comment:
Nice catch!

Reply:
Thank you for testing 👍 That's interesting. That vertical gray is the reason why I kept the CSS. For some reason, passing `icon` property was removed when I visit one of the WC pages.

I'll look into it

Reply:
Just checked it with Brave and it worked fine for me.

![Screen Shot 2021-07-21 at 7 48 40 PM](https://user-images.githubusercontent.com/4723145/126583711-7a57d11c-9cf7-437d-8718-19eac31d2d98.jpg)

The icon also looks identical to WC Pay plugin icon

![Screen Shot 2021-07-21 at 7 49 32 PM](https://user-images.githubusercontent.com/4723145/126583732-c1f7a997-4ff8-4baa-aecc-5365285eb40a.jpg)

Could you run `npm run build` or `npm run start` and check again please?

Reply:
After rebuild and cleared cache, it worked. I think it might've been my cache, sorry!

Reply:
I forgot to mention, the reversed icon can be seen Calypso

![image](https://user-images.githubusercontent.com/3747241/126584106-df21ae34-e698-4f1b-aafd-e2f69b468e9d.png)